askama_shared/src/escaping.rs

```diff
@@ -55,26 +55,33 @@ fn escapable(b: u8) -> bool {
 }
 
 pub fn escape(s: String) -> String {
-    let mut found = Vec::new();
-    for (i, b) in s.as_bytes().iter().enumerate() {
-        if escapable(*b) {
-            found.push(i);
+    let found: usize = if let Some(c) = s.chars().next() {
+        if escapable(c as u8) {
```

Comment:
This seems like it could potentially be wrong, but I don't know enough about the details of how Unicode and the Rust `char` type work to be sure. Can you dig into that and convince me? :smile:

Reply:
With the change from above it is solved.